packages/launchpad/cypress/e2e/config-files-error-handling.cy.ts

```diff
@@ -60,6 +60,7 @@ describe('Config files error handling', () => {
     cy.get('body').should('not.contain.text', 'Cypress no longer supports')
   })
 
+  // NOTE: TODO: (tim) re-enable when we land the error fixes
```

Comment:
What is this comment relate to? I don't see a skip or anything

Reply:
Oh, good call, I think it was failing at one point and I got it passing